README.md

```diff
@@ -392,9 +395,82 @@ function determineComponent(prop) {
   if (basicType === "integer") return componentInteger(prop);
   if (basicType === "array") return componentArray(prop);
   if (basicType === "object") return componentObject(prop);
+  if (basicType === "null") return null
 
   return null;
 }
 ```
 
 Please have a look at a [reference GUI implementation here](/blob/main/reference-gui/src/gdd-gui.jsx), and [its live demo here.](https://superflytv.github.io/GraphicsDataDefinition/reference-gui/dist/)
+
+
+## Playout Options
+
+In the Schema, there is an option to add the `gddPlayoutOptions` object with various properties therein.
+These properties can be read by the playout client in order to modify how it'll display / use / play the template.
+
+_Note: All of the properties inside of `gddPlayoutOptions` are optional._
+
+```typescript
+{
+  "title": "My GFX Template",
+  "type": "object",
+  "properties": {
+    "myProperty": {
+      // ...
+    }
+  },
+  "gddPlayoutOptions": {
+    /** This object contains options that aren't tied to a specific playout server */
+    "client": {
+      /**
+       * The suggested duration of the template (in milliseconds)
+       * -1 means that it is manually taken out
+       * undefined should be treated as -1
+       * (This is ignored if steps=0)
+       * Defaults to -1
+       */
+      "duration": number,
+
+      /**
+       * Number of steps in the template
+       * 1 means that there are no steps (ie there's only "the default step").
+       * 2 or more means that it can be "stepped" (ie 2 means it can be stepped once).
+       * -1 means "infinite" number of steps.
+       * 0 means that the template is "volatile" / "fire and forget" (template really has no duration, like a bumper).
+       * Defaults to 1
+      */
+      "steps": number,
+
+      /**
+       * How the data should be formatted.
+       * This is mostly used for the older CasparCG flash-based xml data format.
+       * Defaults to "json"
+       */
+      "dataformat": "xml" | "json"
```

Comment:
Since this is a JSON Schema, the transformation this JSON object into an XML document needs to be specified. How are JSON Arrays transformed to XML elements? How are element names built? What is an attribute and what is an element text child node?

Reply:
Good point! I'll add a section describing the CasparCG XML format